recipes/google-cloud-kms/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "google-cloud-kms" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2167e1c599ea1d9fed96d317ad817d0b37497460c70f11aafa13a24ede7c9c35
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [win]
```

Can this be `noarch`?
```suggestion
  noarch: python
```
It's okay for Windows to fail due to missing dependences for `noarch` packages. If not, we need to change how Python is constrained.